skin.json

```diff
@@ -64,6 +64,7 @@
   },
   "responsive": {
     "breakpoints": {
+      "audioOnly": {"id": "audioOnly", "name": "oo-xsmall", "multiplier": 0.7},
```

Comment:
Would it make sense for the key to be something like `audio-xs` or `audio-only-xs` in case more audio only breakpoints are needed later on?

Reply:
Will add